### PR TITLE
Initial README for the `derp` emulator experiment

### PR DESCRIPTION
**Pull request by drphilmarshall**
@rmandelb Here's the repo where I will be fiddling with a simple emulator - this initial PR just contains a simple README to explain what I am trying to do. 

Also, check this out: I started a DESC project in a public repo, and tried to follow the Pub Board repo guidelines. I needed to think a bit about the logical ordering of the "Contact" section, which explains to an outsider what restrictions the code is under - I think the result looks sensible, but see what you think.  cc: @digel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
-A first attempt at a simple LSST DRP catalog emulator
+
+A first attempt at a simple LSST DRP catalog emulator. 
+
+A number of planned DESC project will either need, or could use, a wide area mock LSST catalog, including stars, galaxies, and artifacts, with realistic errors and uncertainties, and with the correct anticipated schema, as would be generated by the LSST Facility's Data Release Production (DRP) pipeline and released to the LSST science community. Most applications will need a mock Object table, while some will need mock ForcedDIASource light curves. DC2 will provide a 300 square degree DRP catalog that we can use to train an emulator, that can then take in a much larger "truth" catalog and generate the corresponding wide area mock LSST catalog.
+
+## Limitations
+
+What makes `derp` so simple?
+* We ignore all blended objects.
```
Comment (rmandelb) [inline README.md]: Meaning... we ignore the impact of blending on object detection and properties?  Or something else?  "We ignore all blended objects" could be taken to mean we literally don't emulate anything that could be blended (which to zeroth order means dropping everything - which, now that I think about it, would be a simple and easy-to-implement emulator but not terribly satisfying).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+## Sources
+
+This project builds on:
+- Sean Mclaughlin's CMU 2018 Hack Day project [`blendtec`](https://github.com/mclaughlin6464/blendtec)
+- The [DC2-analysis tutorial notebooks](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/README.rst)
+
+## Contact
+
+Comments, questions, feedback etc welcome, via the [issues](https://github.com/LSSTDESC/derp/issues/new?body=@drphilmarshall).
```
Comment (rmandelb) [inline README.md]: I did not know this trick for getting it to start a new issue addressed at you. Very cool.

**Review by rmandelb — DISMISSED** [review]
Looks good to me!  I had just one question/request for clarification on this initial README (see below).  

I thought you were joking about the name.  :)  Live and learn, I guess.

File: README.md (hunk 1 above)
Comment (drphilmarshall) [reply README.md]:
```suggestion
* We ignore all blended objects, focusing (for now) on one-to-one friends-of-friends matches between the truth table and the DRP Object table.
```

File: README.md (hunk 2 above)
Comment (drphilmarshall) [reply README.md]: Thanks! I'm preparing for a time when I am _not_ reading all LSST DESC GitHub notifications ;-)

File: README.md (hunk 1 above)
Comment (rmandelb) [reply README.md]: Gotcha.  That suggested change looks good to me.

File: README.md (hunk 2 above)
Comment (rmandelb) [reply README.md]: Oh no, will you go into withdrawal??

**Pull request merged**